lib/docs/wasm/Walk.zig

```diff
@@ -537,6 +537,13 @@ fn struct_decl(
     try w.file.get().scopes.putNoClobber(gpa, node, &namespace.base);
     try w.scanDecls(namespace, container_decl.ast.members);
 
+    // TODO: Support for doctests on file-as-a-struct types without using
+    // the filename to find the associated test.
```

Comment:
#363

Reply:
https://github.com/ziglang/zig/issues/23614 is the relevant issue until this is merged. If this is merged, I'll open a separate issue for this TODO.

To be clear, this TODO would be relevant for something like:

```
// Foo.zig

const Bar = @This();

test Bar {
    // ...
}
```

With this PR, this doctest won't be found, since it only looks for `Foo` from the filename.

Reply:
What I meant is that the TODO should be replaced with an issue link.

Reply:
It's awkward because that issue is sort of in limbo--it only needs to exist if this PR is merged.

Happy to open an issue and replace the TODO with a link if this PR is going to be merged.